test/boost/sstable_datafile_test.cc

```diff
@@ -113,10 +113,10 @@ SEASTAR_TEST_CASE(datafile_generation_09) {
         m.set_clustered_cell(c_key, r1_col, make_atomic_cell(int32_type, int32_type->decompose(1)));
         mt->apply(std::move(m));
 
-        auto sst = env.make_sstable(s, tmpdir_path, 9, la, big);
+        auto sst = env.make_sstable(s, tmpdir_path, 9, sstables::get_highest_sstable_version(), big);
```

Comment:
changing la to "highest" makes sense, I won't object, but I think an even better thing would have been to test all writable formats, not just the latest one.
Imagine that next week we add a new format "na" - it will mean that at that point we'll stop testing the still-actively-used mc format.

Reply:
I agree but I think this is outside of the scope of this PR. Expanding tests is probably better suited for a follow-up, no?